Symbol List - Exclude.tmPreferences

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plist version="1.0">
+<dict>
+    <key>scope</key>
+    <string>
+        <!-- 
+             In Fortran, we can write 'type :: myType' and 'end type myType', 
+             and similarly for other entities. This file avoids counting both 
+             instances in the symbol list, so that you are only directed to the 
+             declaration line, e.g. 'type :: myType'.
+         -->
+        source.fortran entity.name.function.fortran - meta.function.declaration.fortran,
+        source.fortran entity.name.class.fortran - meta.class.declaration.fortran
```

Comment:
This line is not necessary if you use the scope `meta.class.declaration entity.name.class.fortran` instead of only `entity.name.class.fortran` in the `Symbol List - Include.tmPreferences` file.

I used this approach to explicitely exclude function names that don't have the `meta.function.declaration` scope, because Sublime Text automatically adds `entity.name.function` to the symbol list for all languages. Therefore `entity.name.function.fortran` in the "Include" file isn't really necessary as well (but of course it isn't harmful either).
Afaik this is the only scope name for which this happens.

Reply:
Thanks for reviewing this! I've made the changes you suggested and will merge. 

I will make some new PRs to get modules, submodules, and program into the symbol list. However, I think we should exclude interfaces, since constructors are given by interfaces with the same name as the class:
```fortran
type :: myType
...
end type myType

interface myType
   procedure :: new_myType
end interface myType
```
With interfaces included, we would get `myType` twice in the symbol list.